scene/gui/tree.cpp

```diff
@@ -3669,7 +3669,7 @@ void Tree::gui_input(const Ref<InputEvent> &p_event) {
 						}
 						if (rect.has_point(mpos)) {
 							if (!edit_selected()) {
-								emit_signal(SNAME("item_icon_double_clicked"));
+								emit_signal(SNAME("item_activated"));
```

Comment:
What's this change for?

Reply:
I'll say an artefact of my earlier changes. The "if (rect.has_point(mpos)) { // code }" checks if we're double clicking on the icon or on the text. The change to item_activated permitted to the scene tree to receive the signal and edit the text or focus the node. It doesn't affect my changes so I'll leave it as it was.